packages/react/package.json

```diff
@@ -49,12 +49,14 @@
     "@svgr/cli": "^6.5.1",
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.8",
+    "email-regex": "^5.0.0",
```

Comment:
the regex packages should probably go into "dependencies" instead of "devDependencies"